packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -171,6 +171,7 @@ export const registerScaledPriceAuthority = async (
     ]),
   ]);
 
+  // FIXME can this work without decimalPlaces? if so, read from boardAux
```

Comment:
I suspect this usage requires `decimalPlaces`.  (cc @michaelfig)

> // FIXME can this work without decimalPlaces? if so, read from boardAux

This seems backwards. If it can work w/o dp, then it should. If it can't, then it needs an alternative source like boardAux.

Reply:
I think my description of "unit amounts" as `decimalPlaces` is causing confusion.  I wrote that description when the only source of "unit amount" information was `getDisplayInfo().decimalPlaces`.  The code below can be made to work without `getDisplayInfo()`, if some alternative like `boardAux` conveys the same information.  But the functionality provided by the code needs some source of "unit amounts".